### PR TITLE
Changelog for 0.27.0

### PR DESCRIPTION
**Pull request by rgrunber**
- Set XML language server to upcoming 0.28.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,32 @@
 # Change Log
 
+## [0.27.0](https://github.com/redhat-developer/vscode-xml/milestone/36?closed=1) (May 27, 2024)
+
+### Enhancements
+
+ * Align XML attributes when formatting. See [#272](https://github.com/redhat-developer/vscode-xml/issues/272).
```
Comment (datho7561) [inline CHANGELOG.md:7]: Add [the setting feature from lemminx](https://github.com/eclipse/lemminx/pull/1644/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR7)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,32 @@
 # Change Log
 
+## [0.27.0](https://github.com/redhat-developer/vscode-xml/milestone/36?closed=1) (May 27, 2024)
```
Comment (datho7561) [inline CHANGELOG.md:3]: Should we release this as 0.28.0 so that lemminx and vscode-xml retain the same minor version?

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (fbricon) [reply CHANGELOG.md:3]: no need

Comment (datho7561) [reply CHANGELOG.md:3]: okay

File: CHANGELOG.md (hunk 1 above)
Comment (rgrunber) [reply CHANGELOG.md:7]: Angelo mentioned that https://github.com/redhat-developer/vscode-xml/issues/945#issuecomment-1836537103 fixed it. I mention it under bug fixes. I could reword it and move it up to enhancements.

Comment (datho7561) [reply CHANGELOG.md:8]: Okay. Sounds good.

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks, Roland!

**Pull request merged**